pkg/operator/controller/externaldns/controller_test.go

```diff
@@ -289,7 +289,7 @@ func testExtDNSInstance() *operatorv1alpha1.ExternalDNS {
 func testSecret() *corev1.Secret {
 	return &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      testSecretName,
+			Name:      "external-dns-credentials-test",
```

Comment:
nit: maybe declare a constant for all literals that are duplicated in the code i.e "deployment"  , "external-dns-test" (3 times) , "serviceaccount" (6 times),  "externaldns" (3 times). It helps when refactoring code (i.e change in one place only) and neatens code

Reply:
Done here 8d680dc2c2f2b12800f60a5326c97e283d80fadc